pkg/errors/errors.go

```diff
@@ -10,15 +10,17 @@ import (
 const (
 	ErrInspectImage int = 1 + iota
```

Comment:
you don't have to do this now, but I think the correct name for all there errors should be:

InspectImageError
PullImageError
...etc